src/xdr/Fuzz-test.x

```diff
@@ -1,15 +1,15 @@
-#pragma once
-
 // Copyright 2015 Stellar Development Foundation and contributors. Licensed
 // under the Apache License, Version 2.0. See the COPYING file at the root
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
 
-#include "util/Logging.h"
+%#include "xdr/Stellar-transaction.h"
 
 namespace stellar
 {
 
-void fuzz(std::string const& filename, el::Level logLevel,
-          std::vector<std::string> const& metrics);
-void genfuzz(std::string const& filename);
+struct FuzzTransactionInput
```

Comment:
you don't need a custom type for this (and the xdr file): all you need to do is generate random operations